Hotfix: Delta 1.5.7.1 Patch

ItsMeJaba opened the pull request:
## What Does This PR Do
Исправляет крупные и не очень недочеты последнего патча, так или иначе влияющие как на игровой процесс, так и на баланс.

## Images of changes
![1 5 7 1](https://user-images.githubusercontent.com/91949115/163205879-451416b2-f8c1-4e7b-a95b-469c20168200.png)


## Changelog
:cl:
# Нерфы:
* Обломки на севере от пермабрига: тулбокс заменен на аварийный(красный), ЕВА и шкаф синдиката на стандартный пожарный шкаф.
* Потайная комната синдиката: тулбокс заменен на фейковый, удален оперативник и пояс (их съела уточка).

# Фиксы:
* Камера в Lounge-зоне прибытия теперь не ливитирует по среди коридора
* Ограждена точка доставки в коморку уборщика мул-ботом
* АПЦ в баре теперь подсоединено к сети
* Скраббер в ксенобиологии подключен
* Скраббер в коморке бармена подключен
* Мусор из ксенобиологии теперь не будет застревать в защитной обшивке
* Коморка синдиката теперь не проветривается вакуумом от 3 фальш-стен
* Мусоропровод на кухне теперь не выбрасывает все в коридор
* Плакат на кухне, что загораживал целый стол удален
* Восстановлен кусок стены шаттла прибытия
* Убраны 2 фальш-стены, потенциально проветривающие техи карго
* Раундстарт спавны снабжения и уборщика теперь работают
* Консоль СБ в прибытии снова работает в сети камер станции

# Нововведения:
* Перепеределан мусоросброс
* Переделаны северо-восточные солнечные панели
* Добавлен шахтерам несколько персональных шкафчиков и их вендор
* Новое, более грязное, оформление зоны доставки карго
* Добавлен АТМ у входа в бриг
* Добавлены гранаты-барьеры в оружейную брига
* Самое важное: добавлена 3 резиновая уточка
/:cl:

ItsMeJaba committed:
Hot-Feat-fix: Delta 1.5.7.1 Patch

# Нерфы:
* Обломки на севере от пермабрига: тулбокс заменен на аварийный(красный), ЕВА и шкаф на стандартный пожарный шкаф.
* Потайная комната синдиката: тулбокс заменен на фейковый, удален оперативник и пояс (их съела уточка).

# Фиксы:
* Камера в Lounge-зоне прибытия теперь не ливитирует
* Ограждена точка доставки в коморку уборщика мул-ботом
* АПЦ в баре теперь подсоединено к сети
* Скраббер в ксенобиологии подключен
* Скраббер в коморке бармена подключен
* Мусор из ксенобиологии теперь не будет застревать в защитной обшивке
* Коморка синдиката теперь не проветривается вакуумом от 3 фальш-стен
* Карго вновь под напряжением со стороны космоса

# Нововведения:
* Перепеределан мусоросброс
* Переделаны северо-восточные солнечные панели
* Добавил шахтерам несколько персональных шкафчиков и их вендор
* Добавлен АТМ у входа в бриг
* Добавлены гранаты-барьеры в оружейную брига

TrashDoxx commented:
Ыы, я Дижея просил залить фиксы.

TrashDoxx commented:
Точка доставки не ошибка. Отсутствие проводов не ошибка.

TrashDoxx commented:
Точка доставки не ошибка. Отсутствие проводов не ошибка.

TrashDoxx commented:
На нерфы похуй. Остальное в порядке.

ItsMeJaba committed:
Фикс фикса

"Отсутствие проводов не ошибка"

ItsMeJaba committed:
New problems

Fix: труба из мусоропровода кухни восстановлена, кусок стены шаттла восстановлен

ItsMeJaba committed:
Cosmetic and something else

fix: Убраны 2 фальш-стены, потенциально проветривающие техи карго
Незначительные косметические решения (состаривание участков тех. тоннелей)

ItsMeJaba committed:
AND NEW FIXES!

FIx: Спавны отдела снабжения теперь работают
И еще немного косметики, теперь уже в карго

TrashDoxx commented:
Так, сча.

TrashDoxx commented:
Чёрт возьми, Жаба, твой форк не влезает в мой ремоут.

TrashDoxx commented:
Мой отец учил меня не стесняться таких ситуаций, но я весьма озадачен.

TrashDoxx commented:
Это был выебон GitKraken'а, а Extensions всё починил. Валлат был прав буквально во всём.

TrashDoxx committed:
More Fixes

Пару турфов да объектов поправил, тю.

ItsMeJaba committed:
Merge pull request #1 from TrashDoxx/delta-1.5.7

More Fixes

TrashDoxx commented:
Дочинены спавнеры, добавлена консоль.

TrashDoxx commented:
@Bizzonium Надо мержить, Жаба закончил, я тоже.

ItsMeJaba committed:
Another fix

Добавил гробиков шахтеров для особенно бунтующих и восстановил сеть камер на консоли СБ в прибытии
